config/sync/message.template.os2loop_message_answer_added.yml

```diff
@@ -9,6 +9,11 @@ label: 'Comment added to post'
 description: message_type
 text:
   -
-    value: 'Der er kommet et nyt svar til spørgsmålet: <a href="/node/[message:field-message-node-reference:nid]">[message:field-message-node-reference:field_description]</a>'
+    value: 'Der er kommet et nyt svar til spørgsmålet: <a href="/node/[message:os2loop_message_node_refer:entity:nid]">[message:os2loop_message_node_refer:entity:title]</a>'
```

Comment:
Can we generate an absolute url to the content here? This may come in handy if we want to send out notification emails in the future.

Reply:
I tried some different tokens, but they all seem relative to page they are on, might be a drupal bug. For now it's left as is, we may need to revisit when email notifications are implemented